captum/attr/_utils/visualization.py

```diff
@@ -230,6 +230,7 @@ def visualize_image_attr(
     plt_axis.yaxis.set_ticks_position("none")
```

Comment:
nit: Can possibly either update the comment above (or add a comment below) that grid lines are disabled?